ERCS/erc-8034.md

```diff
@@ -0,0 +1,769 @@
+---
+eip: 8034
+title: Royalty Standard for Referable NFTs
+description: A standalone royalty standard for Referable NFTs, supporting multiple recipients, reference-based royalty distribution.
+author: Ruiqiang Li (@richard-620) <richard.620.research@gmail.com>, Qin Wang <qin.wang@data61.csiro.au>, Shiping Chen <shiping.chen@data61.csiro.au>, Saber Yu (@OniReimu), Brian Yecies <byecies@uow.edu.au>, John Le <johnle@uow.edu.au>
+discussions-to: Link will be added
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR
https://ethereum-magicians.org/c/ercs/57